components/bpmn/bpmn-rest/src/main/java/org/wso2/carbon/bpmn/rest/service/stats/UserService.java

```diff
@@ -77,23 +84,29 @@ public ResponseHolder getNoOfTasksCompletedByUser() throws UserStoreException {
 
         List listOfUsers = new ArrayList<>();
         ResponseHolder response = new ResponseHolder();
+
         try {
             String[] users = (String[]) getUserList().getData().toArray();
 
-        for (String u : users) {
-            UserTaskCount userInfo = new UserTaskCount();
-            userInfo.setUserName(u);
-
-            long count = BPMNOSGIService.getHistoryService()
-                    .createHistoricTaskInstanceQuery().taskAssignee(u).finished().count();
-            if (count == 0) {
-                userInfo.setTaskCount(0);
-            } else {
-                userInfo.setTaskCount(count);
+            for (String u : users) {
+                UserTaskCount userInfo = new UserTaskCount();
+                userInfo.setUserName(u);
+                String assignee ;
+                if ( tenantDomain == DOMAIN_OF_SUPER_TENANT ) {
```

Comment:
The string comparison is wrong. Is this intentional? seems to be a bug in the code